<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>achievmentItem</class>
 <widget class="QWidget" name="achievmentItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>192</width>
    <height>183</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>191</width>
     <height>181</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout" stretch="1,4">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="dayOfWeek_label">
        <property name="text">
         <string>label 1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="dayOfMonth_label">
        <property name="text">
         <string>label 2</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,4">
        <item>
         <widget class="QLabel" name="habitPixmap_label">
          <property name="text">
           <string>habit pixmap</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="habit_label">
          <property name="text">
           <string>txt</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,4">
        <item>
         <widget class="QLabel" name="regularTaskPixmap_label">
          <property name="text">
           <string>regular pixmap</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="regularTask_label">
          <property name="text">
           <string>txt</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
